zerver/webhooks/dialogflow/doc.md

```diff
@@ -0,0 +1,26 @@
+Get personal message notifications in Zulip for the results of your Dialogflow queries!
+
+1. Next, on your {{ settings_html|safe }}, [create a bot](/help/add-a-bot-or-integration) for
+   {{ integration_display_name }}. Make sure that you select
+   **Incoming webhook** as the **Bot type**.
+
+1. The API key for an incoming webhook bot cannot be used to read messages out
+   of Zulip. Thus, using an incoming webhook bot lowers the security risk of
+   exposing the bot's API key to a third-party service.
+
+1. Construct the URL for the Dialogflow bot using the bot's API key and your Zulip email.
+   The webhook URL should look like:
+
+   `{{api_url}}?api_key=BOT'S_API_KEY&email=foo@example.com`
+
+1. Modify the parameters of the URL above where `api_key` is the API key of your Zulip bot
+   and `email` is your Zulip email.
+
```

Comment:
Two things:
* Combine the 4 points above this comment into one.
* Remove all screenshots except the one at the end.